### PR TITLE
Add meta parameter to action creators

### PR DESCRIPTION
**Pull request by TeaBough**
Thx for this very usefull glue between redux-form and saga ! I need to add meta field to some action created by the action creators from `createFromAction` hence this PR ;-)

### CODE REVIEW

**Comment by jgerlier** [comment]
+1

File: src/index.js [hunk 1]
```diff
@@ -38,10 +39,10 @@ function createFormAction (requestAction, types, payloadCreator = identity) {
     throw new Error('Must include two action types: [ SUCCESS, FAILURE ]');
   }
 
-  return Object.assign((data, dispatch) => {
+  return Object.assign((data, dispatch, meta) => {
```
Comment (mhssmnn) [inline src/index.js]: @TeaBough thanks for the PR! This function typically gets passed as a callback to `handleSubmit`, which then [calls](https://github.com/erikras/redux-form/blob/master/src/handleSubmit.js#L39) it with `values`, `dispatch` and `props`

Comment (TeaBough) [reply src/index.js]: @mhssmnn what about `Object.assign((data, dispatch, props, meta) => { ...` ?

Comment (yakirsobelman) [reply src/index.js]: > @TeaBough thanks for the PR! This function typically gets passed as a callback to `handleSubmit`, which then [calls](https://github.com/erikras/redux-form/blob/master/src/handleSubmit.js#L39) it with **`values`, `dispatch` and `props`**

@mhssmnn The README only states `values` and `dispatch`, so I guess `props` should be added as well. I had a similar issue regarding this one.

**Pull request closed without merging**